go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/celestiaorg/celestia-node
 
-go 1.23.6
+go 1.24.0
+
+toolchain go1.24.3
```

Comment:
We don't need to specify toolchain I think. It should work without it too

Reply:
done